ntoskrnl/include/internal/se.h

```diff
@@ -24,6 +24,19 @@ typedef struct _KNOWN_COMPOUND_ACE
     ULONG SidStart;
 } KNOWN_COMPOUND_ACE, *PKNOWN_COMPOUND_ACE;
 
+typedef struct _ACCESS_CHECK_RIGHTS
+{
+    ACCESS_MASK RemainingAccessRights;
+    ACCESS_MASK GrantedAccessRights;
+    ACCESS_MASK DeniedAccessRights;
+} ACCESS_CHECK_RIGHTS, *PACCESS_CHECK_RIGHTS;
+
+typedef enum _ACCESS_CHECK_RIGHT_TYPE
```

Comment:
Are these a REACTOS-only addition, or do they also exist on some form in Windows as well?

Reply:
These are specific ReactOS-only stuff, on my own personal way.

Reply:
I was thinking about some `#ifdef _REACTOS_`... also the first member of the enum being "...Maximum" sounded unnatural, because usually these "Maximum" are always the last ones in an enum.

Reply:
I'm not sure what you are talking about, that doesn't make sense. This is a private Se header so someone is free to do whatever they want so I dunno what you're trying to convey here.